scrapers/scrape_zg.py

```diff
@@ -49,3 +60,8 @@
     print('ICU:', data[day]['Hospitalisierte in Intensivpflege'])
     print('Recovered:', data[day]['Genesene'])
     print('Deaths:', data[day]['Todesfälle'])
+    date = sc.date_from_text(day).isoformat()
+    if date in data2 is not None and 'Positiv getestete Personen' in data2[date] is not None:
```

Comment:
Where does this come from? I don't see `Positiv getestete Personen` in the CSV and can't see the direct link between this and `isolated`

Reply:
I didn't see that there are two different CSVs, nevermind.